src/main/webapp/WEB-INF/classes/com/entity/user/LoggedInUser.java

```diff
@@ -5,35 +5,48 @@
 import com.entity.review.Review;
 import com.entity.room.Room;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.NoSuchElementException;
 
 public class LoggedInUser implements User {
     private String username;
     private String password;
-    private final Integer id;
+    private final int id;
     private final List<Review> reviews;
-    private final List<Reviewable> favourites;
     private final List<Building> favouriteBuildings;
     private final List<Room> favouriteRooms;
+    private final List<Reviewable> favourites;
 
     /**
      * @param username the users username
      * @param password the users password
      * @param id the users ID
      * @param reviews the list of reviews the user has left
-     * @param favourites the list of all the users favorites
      * @param favouriteBuildings the list of the users favourite buildings
      * @param favouriteRooms the list of the users favourite rooms
      */
-    public LoggedInUser(String username, String password, Integer id, List<Review> reviews, List<Reviewable> favourites, List<Building> favouriteBuildings, List<Room> favouriteRooms) {
+    public LoggedInUser(String username, String password, Integer id, List<Review> reviews, List<Building> favouriteBuildings, List<Room> favouriteRooms) {
```

Comment:
If you want to change id from Integer to int then you have to change all occurences of it. The LoggedInUser constructor still takes in an Integer object for id.